src/models/host.ts

```diff
@@ -0,0 +1,5 @@
+export interface Host {
+  addEventListener: Window['addEventListener'];
```

Comment:
Can we use `typeof window.addEventListener` instead of `Window['addEventListener']`?

Reply:
We could, but as fas as I know `Window['addEventListener']` is the recommended way.